docs/whats-new.md

```diff
@@ -87,6 +87,10 @@ _Benchmarks ran using scripts on a 2012 MacBook Pro, 2.3 GHz Intel Core i7, 8 GB
 
 - Upgraded `apache-arrow` version to 4.0.0
 
+**@loaders.gl/terrain**
+
+- Added new `tesselator` option in `TerrainLoader` for selecting desired method of mesh reconstruction
```

Comment:
Maybe mention we support both TIN (Triangular irregular networks) and Delaunay triangulation.

Reply:
Isn't delaunay triangulation a strict subset of TINs?

Reply:
I haven't read up enough to say, but perhaps you can propose better wording? 

I just would be nice to articulate the differences in a functional way, rather than referencing two particular js modules.

Reply:
Maybe just

```suggestion
- Added new `tesselator` option in `TerrainLoader` for selecting desired method of mesh generation. Use `'martini'` for faster but coarser mesh generation, or `'delatin'` for a slower method that supports non-square images.
```